api_server.py

```
@@ -75,94 +82,111 @@ def format_hours(minutes: Optional[int]) -> Optional[str]:
-            "end_time_display": display,
-        }
+    return {
+        "available": True,
+        "duration": format_hours(duration_min),
+        "end_time_display": format_availability_display(end_time, now),
+    }
 
 
 def get_weekly_stats(crew_id: int) -> Dict:
     start, end = get_week_boundaries()
     now = datetime.now()
```

Ruff (0.15.5)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)